zerver/tests/test_push_notifications.py

```diff
@@ -1567,6 +1567,78 @@ def test_user_message_does_not_exist(self) -> None:
             )
             mock_push_notifications.assert_called_once()
 
+    def test_latex_math_formulas_in_push_notification(self) -> None:
+        # TODO: Fix the following test case when content is rendered properly in the push notification.
```

Comment:
I don't understand the TODO comment here -- are you still trying to figure out what code needs to change?

`get_mobile_push_content` is the function you probably want.

Reply:
I saw that function but I was unable to find the possible fix for that.

Reply:
Can you provide more details in a "#backend" thread?